l1-contracts/contracts/zksync/facets/Executor.sol

```diff
@@ -26,6 +27,10 @@ contract ExecutorFacet is Base, IExecutor {
 
     bool constant VALIDIUM_MODE = $(VALIDIUM_MODE);
 
+    function getPubdataPriceMode() external view returns (PubdataPricingMode) {
```

Comment:
As @StanislavBreadless mentions [here](https://github.com/matter-labs/zksync-era/pull/1015#discussion_r1481252548), this getter should be marked in doc comments as unstable, since it'll likely change in the future.

Reply:
Done in [3fd6f9e](https://github.com/matter-labs/era-contracts/pull/224/commits/3fd6f9ee23ffeb0ac0d498d369f40161941d69ce) and [6da63cc](https://github.com/matter-labs/era-contracts/pull/224/commits/6da63cc98fec09a52af766e5f3d76044b8198536)